CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.4.0
+
+- Fixed an issue where the action would get stuck at 'Getting current replicas' for apps requesting more than one replica
```

Comment:
```suggestion
- Fixed an issue where the action would get stuck at 'Getting current replicas' for apps requesting more than one replica
- Does a health check every 10 seconds instead of increasing the wait time exponentially. Times out now after 10 minutes.
```